lib/kvdb/kvdb_rparams.c

```diff
@@ -1106,6 +1106,44 @@ static const struct param_spec pspecs[] = {
             },
         },
     },
+    {
+        .ps_name = "cn_maint_threads",
+        .ps_description = "max number of cn maintenance threads",
+        .ps_flags = PARAM_FLAG_EXPERIMENTAL,
+        .ps_type = PARAM_TYPE_U32,
+        .ps_offset = offsetof(struct kvdb_rparams, cn_maint_threads),
+        .ps_size = PARAM_SZ(struct kvdb_rparams, cn_maint_threads),
+        .ps_convert = param_default_converter,
+        .ps_validate = param_default_validator,
+        .ps_default_value = {
+            .as_uscalar = 17,
+        },
+        .ps_bounds = {
+            .as_uscalar = {
+                .ps_min = 1,
+                .ps_max = 256,
```

Comment:
Same comment about u8 here.

Reply:
Yep, I'll make them u16, would save some more space in the rparams record..